docs/frequently-asked-questions.rst

```diff
@@ -216,6 +190,74 @@ No, a function call from one contract to another does not create its own transac
 you have to look in the overall transaction. This is also the reason why several
 block explorer do not show Ether sent between contracts correctly.
 
+=======
+What is the ``memory`` keyword? What does it do?
```

Comment:
Did you re-add this?

Reply:
Not intentionally, the rebase was confusing.